src/tests/mocks/factories.ts

```diff
@@ -21,6 +24,15 @@ const schemaFactory = (): VernaJSONSchemaType => ({
   type: 'object',
 });
 
+const uiSchemaFactory = (): UiSchema => ({
+  testSection: {
+    field1: {
+      'ui:widget': 'numberWidget',
+    },
+    'ui:order': ['select'],
+  },
+});
```

Comment:
Instead of create several factories for uiSchema, you should create only factory which take an argument to be able to override extend and overrides the schema returned.

```ts
const uiSchemaFactory = (uiSchema: Partial<UiSchema>): UiSchema => _.merge({
  testSection: {
    field1: {
      'ui:widget': 'numberWidget',
    },
    'ui:order': ['select'],
  },
}, schema));
```